_config.yml

```diff
@@ -34,12 +34,21 @@ authors:
     description: "Co-Founder of The Library by Stream Sounds"
 
 # Plugins
+whitelist: [jekyll-analytics]
```

Comment:
What does whitelisting do?

Reply:
by default some plugins are blocked by github pages, whitelisting allows the plugins to be used. That is what was gathered through some of the issues in the plugin repo.